core/modules/date/date.field.inc

```diff
@@ -300,6 +300,20 @@ function date_field_widget_info() {
   // Add the fieldset option for select fields only.
   $info['date_select']['settings']['no_fieldset'] = FALSE;
 
+  // HTML5 widget with slightly different settings.
+  $info['date_h5widget'] = array(
```

Comment:
None of the other widgets have `widget` in their machine name, and also I'm not a huge fan of shortening things to a point where they become vague (referring to the `h5` bit). Could we please rename this to either `date_html5_widget` or plain `date_html5`?:

Reply:
Sure, but before I rename everything across all code, I'm still hoping for some core committer feedback re the overall implementation.